src/components/ListItem.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import Checkbox from '../components/Checkbox';
 
-const ListItem = ({ item, shoppingList }) => (
-  <li className="list__item">
+const ListItem = ({ index, item, shoppingList }) => (
+  <li className={index} aria-label={index}>
+    {' '}
```

Comment:
What's the purpose of having this? `{' '}`?

Reply:
That's probably a typo, we'll clean it up, thanks

Reply:
Cleaned!